kakao/src/main/kotlin/com/agoda/kakao/Matchers.kt

```diff
@@ -262,6 +263,50 @@ class DrawableMatcher(@DrawableRes private val resId: Int = -1, private val draw
     }
 }
 
+class TextInputLayoutHintEnabledMatcher(private val enabled: Boolean) : TypeSafeMatcher<View>() {
```

Comment:
Shouldn't we enforce the type to be `TextInputLayout` instead of `View`?

Reply:
Actually, if we will enforce type, project will not compile, since `ViewAssertions.matches(Matcher)` will not accept it as an correct argument.